postorms/Paruek_the_Strong.pl

```diff
@@ -1,5 +1,27 @@
 #Notifies Jeplak_Lord_of_Srerendi that npc was killed in the camp.
+sub EVENT_AGGRO {
+    quest::settimer("tumadds",55);
+}
 
 sub EVENT_DEATH_COMPLETE {
-  quest::signal(210179,0);
-}
\ No newline at end of file
+    quest::signal(210179,0);
+    quest::stoptimer("tumadds");
+}
+
+sub EVENT_TIMER {
+  if($timer == "tumadds") {
+    if($npc->IsEngaged()) {
+        quest::spawn2(210247,0,0,($x + 10),$y,$z,$h);
+        quest::spawn2(210247,0,0,($x + 10),$y,$z,$h);
+        quest::spawn2(210247,0,0,($x + 10),$y,$z,$h);
+        quest::spawn2(210247,0,0,($x + 10),$y,$z,$h);
+        quest::spawn2(210247,0,0,($x + 10),$y,$z,$h);
+        quest::spawn2(210247,0,0,($x + 10),$y,$z,$h);
+        quest::spawn2(210247,0,0,($x + 10),$y,$z,$h);
+        quest::spawn2(210247,0,0,($x + 10),$y,$z,$h);
+        quest::spawn2(210247,0,0,($x + 10),$y,$z,$h);
+        }
+    } else {
```

Comment:
think you've got an extra ` } `
As it stands, your `else` is against `if($timer == "tumadds") {`
If a timer event goes off that isn't `tumadds`, `stoptimer("tumadds")`
Assume you mean, if the npc isn't engaged, then stoptimer.

(this is in lua, there is probably a pl version, or easy to convert)
To make that more explicit, you can use `event_combat(e)` (instead of event_aggro) which has `e.joined`
then you can do something like
```
function event_combat(e)
  if(e.joined) then
    eq.set_timer("tumadds",55);
  else
    eq.stop_timer("tumadds");
  end
end
```